proposals/digest.md

```diff
@@ -0,0 +1,45 @@
+# OCI go-digest project proposal
+
+## Abstract
+opencontainers/image-spec#486 introduces a dependency on a stable upstream implementation of https://github.com/docker/go-digest, which was recently broken out of the https://github.com/docker/distribution project.
```

Comment:
GitHub [doesn't render][1] references like `opencontainers/image-spec#486` in its filesystem browser.  You probably want:

    [image-spec#486][image-spec-486] introduces…

    [image-spec-486]: https://github.com/opencontainerrs/image-spec/pull/486

[1]: https://github.com/opencontainers/tob/blob/24101646d5486b86ace4efbf229bd047b61bbb3a/proposals/digest.md